config/settings.yml

```diff
@@ -472,7 +472,12 @@ flipper:
     - travis.hilton@oddball.io
     - zurbergram@gmail.com
     - regarr@gmail.com # Robin Garrison - AdHoc
-
+    - 593460@bah.com
+    - sonntag_adam@bah.com
+    - roth_matthew@bah.com
+    - Shawkey_Daniel@bah.com
+    - devinmccurdy@gmail.com
```

Comment:
Can you put in the existing list alphabetically please?

Reply:
sorted